easybuild/easyconfigs/s/SWIG/SWIG-3.0.11-GCC-5.4.0-2.26-Python-2.7.12.eb

```diff
@@ -0,0 +1,21 @@
+name = 'SWIG'
+version = '3.0.11'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.swig.org/'
+description = """SWIG is a software development tool that connects programs written in C and C++ with
+ a variety of high-level programming languages."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+toolchainopts = {'pic': True, 'opt': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['d9031d531d7418829a54d0d51c4ed9007016b213657ec70be44031951810566e']
+
+dependencies = [
+    ('Python', '2.7.12', '-bare'),
```

Comment:
@vanzod We usually only use non-bare Python installation as (non-build) dependencies, although I understand this is done because of the `GCC` toolchain you are using.

However, this will be incompatible with anything that uses a (non-bare) Python as dependency...

Isn't that an issue? If not, fine, this can go in.

Reply:
@boegel Since the bare installation is a stripped down version of the non-bare, I doubt there can be compatibility issues with other software that use a non-bare Python as long as the version is the same. I have never tested it though...

Reply:
Well, you won't be able to load the modules together, since they're both `Python`?

Reply:
True. I will go ahead and close this PR until a lower toolchain `Python` will be available.